scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/LiveJobs.java

```diff
@@ -640,6 +640,7 @@ void taskStarted(InternalJob job, InternalTask task, TaskLauncher launcher, Stri
 
         // set the different informations on task
         job.startTask(task);
+        jobData.job.incrementNumberOfNodes();
```

Comment:
unfortunately, it's more complicated than this, as tasks can use multiple nodes

Reply:
you can use something like

`jobData.job.increaseNumberOfNodesUsed(task.getExecuterInformation().getNodes().getTotalNumberOfNodes())`